.gitignore

```diff
@@ -5,3 +5,4 @@ coverage/
 test/browser.html
 site/
 docs/
+.idea/
```

Comment:
Editor artifacts should be part of your [own local gitignore](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)